src/types/express.locals.ts

```diff
@@ -40,33 +40,38 @@ export type SpcpLocals =
   | { uinFin: string; userInfo: string }
   | { [key: string]: never } // empty object
 
-export interface EmailAutoReplyField {
+export type EmailRespondentConfirmationField = {
   question: string
   answerTemplate: string[]
 }
 
-export interface EmailJsonField {
+export type EmailDataCollationToolField = {
   question: string
   answer: string
 }
 
-export interface EmailFormField {
+export type EmailAdminDataField = {
   question: string
   answer: string
   fieldType: BasicField
   answerTemplate: string[]
```

Comment:
what do you think it will take to avoid duplicating both `answer` and `answerTemplate`?

Reply:
Seems to me that this is presentation logic, so splitting of answer into answerTemplate could be done at the last step instead of maintaining the answerTemplate state? But perhaps can tackle separately :)